.prettierrc.cjs

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  embeddedLanguageFormatting: "off",
```

Comment:
Was there a reason to disable this? Seems like it might be marginally useful in README, CONTRIBUTING, etc. docs

Reply:
Oh interesting - I guess prettier does .md? The reason I turned it off is because we have some html string literals that are getting compressed. See [packages/mongodb-artifact-generator/src/operations.ts](https://github.com/mongodb/chatbot/blob/main/packages/mongodb-artifact-generator/src/operations.ts#L87-L109) for example: this would strip the newlines and put everything into one block.

Reply:
Ah very annoying there so probably a good change

Reply:
Also I'm not sure that prettier is what handles the formatting within md files. I wasn't able to get embedded languages formatted either way.